### PR TITLE
Add padding to bottom of text so large letters on't get cut off

### PR DESCRIPTION
**Pull request by petedejoy**

### CODE REVIEW

File: src/instruments/menus/Level1/styles.scss [hunk 1]
```diff
@@ -20,15 +20,15 @@
     overflow: hidden;
     max-width: 22vw;
     text-overflow: ellipsis;
-  }
```
Comment (schnie) [inline src/instruments/menus/Level1/styles.scss]: @petedejoy don't we still want this ending `}`?

File: src/instruments/menus/Level1/styles.scss [hunk 2]
```diff
@@ -48,3 +48,4 @@
   background: $lightGray;
   border: none;
 }
+}
```
Comment (schnie) [inline src/instruments/menus/Level1/styles.scss]: @petedejoy typically we always want a newline at the end of any source code files. This little indicator is saying that it was removed. You should be able to configure your editor to add newlines on save - `    "files.trimTrailingWhitespace": true` in vs code settings.

File: src/instruments/menus/Level1/styles.scss [hunk 3]
```diff
@@ -20,15 +20,15 @@
...
     max-width: 22vw;
     text-overflow: ellipsis;
-  }
+    padding-bottom: 1px;
   &.active .text {
     max-width: 50vw;
   }
 }
 
 .arrow {
```
Comment (schnie) [inline src/instruments/menus/Level1/styles.scss]: @petedejoy is this the little down arrow for the dropdown menu? Did changing this height actually help?

File: src/instruments/menus/Level1/styles.scss (hunk 2 above)
Comment (petedejoy) [reply src/instruments/menus/Level1/styles.scss]: Ah got it- my bad, I guess my linter doesn't catch it in css files so it didn't yell at me. Lemme fix it up.

**Pull request merged**